src/plonemeeting/portal/core/browser/manage_users.py

```
@@ -236,19 +261,27 @@ def handleSave(self, action):
+                    password,
+                    properties={
+                        "email": email,
+                        "username": username,
+                        "fullname": fullname,
+                    },
                 )
                 self.update_user_groups(username, groups_to_assign)
                 self.registration.registeredNotify(username)
                 self.messages.add(_("msg_user_created"), type="info")
-            except Exception as e:
+            except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)